cppapi/server/except.cpp

```diff
@@ -45,6 +45,19 @@ namespace Tango
 char Except::mess[256];
 omni_mutex Except::the_mutex;
 
+std::ostream &operator<<(std::ostream &ss, const Tango::DevFailed &devFailed)
+{
+    for (int i = 0, size = devFailed.errors.length(); i < size; ++i)
+    {
+        DevError error = devFailed.errors[i];
```

Comment:
Consider using reference instead of a copy. `DevError` has CORBA string members which allocate and copy memory from their copy-ctors.

Reply:
ok.